src/component/metrics/jhi-jvm-memory.component.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ Copyright 2013-2017 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://jhipster.github.io/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+import {Component, Input} from '@angular/core';
+
+@Component({
+    selector: 'jhi-jvm-memory',
+    template: `<h4 jhiTranslate="metrics.jvm.memory.title">Memory</h4>
+    <div *ngIf="!updating">
+        <div *ngFor="let entry of jvmMemoryMetrics | keys">
+                <span *ngIf="entry.value.max != -1; else other">
+                    <span>{{entry.key}}</span> ({{entry.value.used / 1048576 | number:'1.0-0'}}M / {{entry.value.max / 1048576 | number:'1.0-0'}}M)
+                </span>
+            <div>Committed : {{entry.value.committed / 1048576 | number:'1.0-0'}}M</div>
```

Comment:
Where does the `1048576` come from ? I think it's to convert bytes?
It would be better to use a custom pipe for this kind of operation (easier to maintain).
https://angular.io/guide/pipes#custom-pipes

Reply:
This is not very important so I'm merging without the pipe -> @clement26695 can you do this later? At least use a constant with an explanation